index.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="css/styles.css">
+
+</head>
+<body>
+    
+
+    <header class="header">
+        <div class="header-links">
+            <ul>
+                <li>
+                    <i id="facebook-icon" class="fa fa-facebook"></i>
+                </li>
+                <li>
+                    <i id="twitter-icon" class="fa fa-twitter"></i>
+                </li>
+                <li>English</li>
+                <li>My Page</li>
+                <li>Logout</li>
+            </ul>
+        </div>
+        <div class="headline-section-nav">   
+            <a href="index.html" class="link logo"><img src="images/cc-logo-dark.png" class="logo-img" alt="logo"></a>
+            <button class="hamburger"><i class="fa fa-bars"></i></button>
+            <nav class="">
+                <ul class="mainMenu d-none">
+                    <li class="closeMenu">&times;</li>
+                    <li><a href="#" class="nav-links">Home</a></li>
+                    <li><a href="about.html" class="nav-links">About</a></li>
+                    <li id="nav-link-border">CC Campaign</li>
+                
+                </ul>
+            </nav>
+       </div>
+
+        <section class="headline-section">
+
+            <div class="headline-section-info">
+                <h1 id="intro-heading">"Hello! Sharing World"</h1>
+                <h2 id="intro-info">CREATIVE COMMONS <br>GLOBAL SUMMIT <br>2015</h2>
+                <p id="intro-description">A joyful celebration believe in the value of openess and sharing, creative a positive change with people from 80 countries is taking place in october, in Korea.</p>
+                <h3 id="intro-date">2015.10.15(THU)~16(FRI)</h3>
+                <p id="intro-address">@ National Museum of Korea, Art Center Nabi and more</p>
+            </div>
+        </section>
+
+    </header>
+
+    <main>
+        <section class="main-program">
+            <div class="main-program-text">
+                <h1>Main Program</h1>
```

Comment:
- [ ] Please it would be better to only one `<h1>`tag for better HTML structure and also won't confuse the google bot